openjdk/variant-oracle.md

```diff
@@ -1,5 +1,5 @@
-## `%%IMAGE%%:<version>` (from 12 onwards), `%%IMAGE%%:<version>-oracle` and `%%IMAGE%%:<version>-oraclelinux7`
+## `%%IMAGE%%:<version>` (from 12 onwards), `%%IMAGE%%:<version>-oracle` and `%%IMAGE%%:<version>-oraclelinux8`
```

Comment:
There are still `oraclelinux7` based images. :thinking: Perhaps we drop the mentioning of `oraclelinux7` and `oraclelinux8` in this heading and provide links to both 7 and 8 in the first paragraph below?

Reply:
Ok, fixed.